### PR TITLE
Add minetest.player_exists()

### PR DESCRIPTION
**Pull request by rubenwardy**
Tested
Credit to @adrido @SmallJoker #4942

### CODE REVIEW

File: builtin/game/misc.lua [hunk 1]
```diff
@@ -114,6 +114,10 @@ function core.get_connected_players()
 	return temp_table
 end
 
+function minetest.player_exists(name)
+	return minetest.get_auth_handler().get_auth(name) ~= nil
```
Comment (nerzhul) [inline builtin/game/misc.lua]: if i remember this call is used in some code parts, no ? if yes can you call it ?

Comment (rubenwardy) [inline builtin/game/misc.lua]: I'm not sure what you mean, please rephrase

Comment (sofar) [inline builtin/game/misc.lua]: I don't see anything wrong with this...

**Comment by sofar** [comment]
Simple enough, it seems.

:+1:

File: builtin/game/misc.lua (hunk 1 above)
Comment (nerzhul) [inline builtin/game/misc.lua]: i mean, if there is similar function in other builting parts, replace it with a call to this function

**Pull request merged**